nixos/modules/services/x11/desktop-managers/plasma5.nix

```diff
@@ -184,15 +183,19 @@ in
 
   config = mkMerge [
     (mkIf cfg.enable {
-      services.xserver.desktopManager.session = singleton {
-        name = "plasma5";
-        bgSupport = true;
-        start = startplasma;
-      };
+      # Seed our configuration into nixos-generate-config
+      system.nixos-generate-config.desktopConfiguration = ''
+        # Enable the Plasma 5 Desktop Environment.
+        services.xserver.enable = true;
+        services.xserver.displayManager.sddm.enable = true;
+        services.xserver.desktopManager.plasma5.enable = true;
+      '';
```

Comment:
I believe this shouldn't be here (this is handled by 99d511124620cad08e1bac2c0ca29ea3222d7f5d, see [here](https://github.com/NixOS/nixpkgs/commit/99d511124620cad08e1bac2c0ca29ea3222d7f5d#diff-03fbf9f534327a96ff2a6a1b6b3aa40a397d3dd9cac950f5af0ed500316b7512)).

Reply:
What? It seems we've pushed removing this and my pull request added this. Without this there the generated config for the plasma5 iso won't have this configuration anymore. Please re add @pasqui23 ?

Reply:
@worldofpeace This PR is based on a Nixpkgs commit from October, before you implemented `system.nixos-generate-config.desktopConfiguration` in November, and hasn't been rebased since. This part of your commit was `git add`ed in this  unrelated PR by mistake.

Reply:
how confusing. github :grin: